modules/EasyAbp.EShop.Stores/src/EasyAbp.EShop.Stores.Application.Shared/EasyAbp/EShop/Stores/Permissions/StoreOwnerAuthorizationExtensions.cs

```diff
@@ -1,50 +1,47 @@
-﻿using Microsoft.AspNetCore.Authorization;
-using System;
+﻿using System;
 using System.Threading.Tasks;
-using Volo.Abp.Authorization;
+using Microsoft.AspNetCore.Authorization;
 
 namespace EasyAbp.EShop.Stores.Permissions
 {
     public static class StoreOwnerAuthorizationExtensions
     {
-        public static Task<bool> IsStoreOwnerGrantedAsync(this IAuthorizationService authorizationService,
+        public static Task<bool> IsCurrentUserStoreOwnerAsync(this IAuthorizationService authorizationService,
```

Comment:
This method's name `IsCurrentUserStoreOwnerAsync` seems to have nothing to do with `policyName`.